build.gradle

```diff
@@ -772,6 +775,37 @@ distributions {
     }
   }
 }
+
+ospackage {
+  packageName = distributions.main.baseName
```

Comment:
this seems to be handled automagically by adding the classifier/without-hadoop option checks in `distRpm` and `distDeb` targets :-)